README.md

```diff
@@ -1,3 +1,52 @@
 # ClusterFuzzLite
+ClusterFuzzLite is a continuous fuzzing solution that runs as part of
+[Continuous Integration(CI)](https://en.wikipedia.org/wiki/Continuous_integration)
+workflows to find vulnerabilities faster than ever before.
+With just a few lines of code, GitHub users can integrate ClusterFuzzLite into
+their workflow and fuzz pull requests to catch bugs before they are committed.
 
-This is a work in progress. More details will be announced soon.
+ClusterFuzzLite is based on [ClusterFuzz](https://google.github.io/clusterfuzz/).
+
+## Features
+
+- Quick code change (pull requests) fuzzing to find bugs before they land
```

Comment:
Delete the S on pull requests for consistency--not something you introduced but I just noticed it